WoW-Realm-Tracker/ViewControllers/FavoritesViewController.swift

```diff
@@ -83,8 +84,8 @@ class FavoritesViewController: UITableViewController {
 }
 
 extension FavoritesViewController: RealmsDelegate {
-    func receivedRealms(realms: [Realm]) {
-        controller = FavoriteRealmsController(realms: realms)
+    func receivedRealms(response: Result<[Realm], NSError>) {
+        response.map { realms in self.controller = FavoriteRealmsController(realms: realms) }
```

Comment:
yep, that's my bad. I didn't realize this was a `Result`. Do `if let controller = controller.value` instead. That will pull the optional success case out.